.ci/test-integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+. "$(dirname $0)/variables.sh"
+
+set -e
+
+TESTS=$(go test -test.v -test.tags=integration -test.short ./integration | grep SKIP | cut -d ' ' -f 3)
```

Comment:
:(

Do you always need the `test.` prefix? I think you can just do `go test -v -tags=integration ...`